### ISSUE TITLE
Suggestion of `useless_attribute` breaks rustfix

### ISSUE THREAD

**Issue by flip1995** [rust-lang/rust-clippy#5021]
Minimal reproducer:

```rust
fn foo() {
    #[allow(clippy::almost_swapped)]
    use std::collections::HashMap;

    let _ = HashMap::default();
}
```

produces 
```
error: useless lint attribute
 --> src/main.rs:2:1
  |
2 |     #[allow(clippy::almost_swapped)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: if you just forgot a `!`, use: `#![allow(clippy::almost_swapped)]`
  |
  = note: `#[deny(clippy::useless_attribute)]` on by default
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#useless_attribute
```

The suggestion in itself is correct. This trips rustfix though, since rustfix removes indentations of suggestions and then slices the text pointed at from the suggestion. Relevant rustfix code can be found here: https://github.com/rust-lang/rustfix/blob/715eff0b15b6ffc5adcb016b6713132159681723/src/lib.rs#L95-L113

[Playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=11ecb4f0a8ffcf97c418bb8e32ebb12e)

**Comment by flip1995** [comment]
I'll assign this issue to myself. If I shouldn't get to open a PR about this though, it's up for grabs.
